.azure-pipelines/steps/install-dotnet-5-sdk.yml

```diff
@@ -0,0 +1,7 @@
+steps:
+- task: UseDotNet@2
+  displayName: install dotnet core sdk 5
+  inputs:
+    packageType: sdk
+    version: $(dotnetCoreSdk5Version)
```

Comment:
It could be, but as it's always the same, we just define it at the top level here https://github.com/DataDog/dd-trace-dotnet/blob/andrew/ultimate-nuke-2/.azure-pipelines/ultimate-pipeline.yml#L19 and it filters through. If it's undefined, this would break, so makes things _slightly_ less verbose